### PR TITLE
Story page

### LINKED ISSUE #37: Story Page

### PR DESCRIPTION
**Pull request by aspensantos**
samplestorydrawer

Closes #37 

Worked off of #107

### CODE REVIEW

File: ui/src/components/StoryDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  country,
+  currentCity,
+  content,
+  state,
+  setState,
+}: StoryDrawerProps): JSX.Element {
+  return (
+    <Drawer
+      anchor="right"
+      open={state == StoryDrawerState.Open}
```
Comment (AbhiPrasad) [inline ui/src/components/StoryDrawer.tsx]: let's do `===` here

**Review by AbhiPrasad — APPROVED** [review]

File: ui/src/components/StoryDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+const StyledRoot = styled.div`
+  width: 100vh;
+  padding-left: 30vh;
+  padding-right: 30vh;
+`;
+
+const StyledFlagImage = styled.img`
+  border-radius: 10px;
+  width: 100%;
```
Comment (hanlinc27) [inline ui/src/components/StoryDrawer.tsx:50]: Should be constrained to 50px by 35px

File: ui/src/components/StoryDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  setState,
+}: StoryDrawerProps): JSX.Element {
+  return (
+    <Drawer
+      anchor="right"
+      open={state === StoryDrawerState.Open}
+      onClose={() => setState(StoryDrawerState.Closed)}
+    >
+      <StyledIconButton onClick={() => setState(StoryDrawerState.Closed)}>
+        <ArrowForwardIcon />
```
Comment (hanlinc27) [inline ui/src/components/StoryDrawer.tsx:86]: This is super tiny but I think the arrow in Figma isn't MaterialUI (it's a bit longer) but I don't think that matters.

File: ui/src/components/StoryDrawer.tsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+const StyledImage = styled.img`
+  border-radius: 0px;
+  width: 100%;
+`;
+
+const StyledRoot = styled.div`
+  width: 100vh;
+  padding-left: 30vh;
+  padding-right: 30vh;
+`;
```
Comment (hanlinc27) [inline ui/src/components/StoryDrawer.tsx:46]: Bottom below story body text should have same padding as top

File: ui/src/components/StoryDrawer.tsx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+            <DrawerAuthorText>{author}</DrawerAuthorText>
+          </Grid>
+          <Grid
+            item
+            xs={6}
+            alignItems="flex-end"
+            alignContent="flex-end"
+            justify="flex-end"
+          >
+            <DrawerRightText>{currentCity}</DrawerRightText>
```
Comment (hanlinc27) [inline ui/src/components/StoryDrawer.tsx]:
```suggestion
            <DrawerRightText>{currentCity}, Canada</DrawerRightText>
```

**Review by hanlinc27 — APPROVED** [review]

File: ui/src/components/StoryDrawer.tsx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    >
+      <StyledIconButton onClick={() => setState(StoryDrawerState.Closed)}>
+        <ArrowForwardIcon />
+      </StyledIconButton>
+      <StyledRoot>
+        <Grid container spacing={2} alignItems="center">
+          <Grid item xs={12}>
+            <DrawerTitleText>{title}</DrawerTitleText>
+          </Grid>
+          <Grid item xs={2}>
```
Comment (hanlinc27) [inline ui/src/components/StoryDrawer.tsx]:
```suggestion
          <Grid item xs={1}>
```

File: ui/src/components/StoryDrawer.tsx [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+            <StyledFlagImage alt="flag" src={ChinaFlag} />
+          </Grid>
+          <Grid item xs={4}>
```
Comment (hanlinc27) [inline ui/src/components/StoryDrawer.tsx]:
```suggestion
          <Grid item xs={5}>
```

File: ui/src/components/StoryDrawer.tsx (hunk 4 above)
Comment (aspensantos) [reply ui/src/components/StoryDrawer.tsx:46]: added 10vh to the bottom padding

File: ui/src/components/StoryDrawer.tsx (hunk 3 above)
Comment (aspensantos) [reply ui/src/components/StoryDrawer.tsx:86]: Just left this the same

File: ui/src/components/StoryDrawer.tsx (hunk 2 above)
Comment (aspensantos) [reply ui/src/components/StoryDrawer.tsx:50]: Used the grid size suggestion you had instead

**Pull request merged**